library/consul

```diff
@@ -1,6 +1,7 @@
 # maintainer: James Phillips <james@hashicorp.com> (@slackpad)
 
-latest: git://github.com/hashicorp/docker-consul@470868df3885ad93f45a2c63c648bf119a544fa4 0.X
+latest: git://github.com/hashicorp/docker-consul@2c2873f9d619220d1eef0bc46ec78443f55a10b5 0.X
+0.7.1: git://github.com/hashicorp/docker-consul@2c2873f9d619220d1eef0bc46ec78443f55a10b5 0.X
 0.7.0: git://github.com/hashicorp/docker-consul@470868df3885ad93f45a2c63c648bf119a544fa4 0.X
 v0.7.0: git://github.com/hashicorp/docker-consul@470868df3885ad93f45a2c63c648bf119a544fa4 0.X
```

Comment:
That's correct - we just wanted to not break existing users. I'll go ahead an PR removing the "v" version.